packages/server/src/arpa_reporter/routes/users.js

```diff
@@ -67,7 +67,13 @@ router.post('/', requireAdminUser, async function (req, res, next) {
       const updatedUser = await createUser(user)
       res.json({ user: updatedUser })
 
-      void sendWelcomeEmail(updatedUser.email, req.headers.origin)
+      let origin = req.headers.origin;
+
+      if (req.headers.referer.indexOf('arpa_reporter')) {
```

Comment:
nit: this route is ARPA Reporter-specific anyway, so this referer check is not necessary

Reply:
also, if you are going to keep it, you need to check `!== -1` because `-1` is truthy

Reply:
also, does this NPE if there is no Referer header? Might need an optional chain

Reply:
great point, I'll go ahead and remove this conditional check entirely. I was wondering whether the `arpa-annual-reporter` endpoint uses the same route so wanted add some type of `if/else` logic.

Reply:
@mattbroussard can you re-review I have resolved the issue by removing the unnecessary conditional.

Reply:
>I was wondering whether the arpa-annual-reporter endpoint uses the same route
That tool is only a single page and doesn't have its own user admin page etc. -- it just uses GOST's.

>can you re-review
fwiw, "approved with comments" generally means the comments are minor enough that reviewer trusts you to make the changes and merge without another review round (still feel free to ask for additional review if you substantially changed the PR but this change is pretty minor)